src/ContentRepository/Packaging/IInstallerFactory.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+
+// ReSharper disable once CheckNamespace
+namespace SenseNet.Packaging
+{
+    /// <summary>
+    /// Defines methods for providing information of a sensenet install package.
+    /// </summary>
+    public interface IInstallerFactory
```

Comment:
Why IInstallerFactory? In general, a factory products something. This interface why defines a factory? Implementation - by the defined two properties - is why an installer?